azure-pipelines.yml

```diff
@@ -18,66 +18,64 @@ jobs:
 
   # Install NuGet
   - task: NuGetToolInstaller@0
-    displayName: Install NuGet 5.6.0
+    displayName: Install NuGet 6.0
     inputs:
-      versionSpec: 5.6.0
+      versionSpec: 6.0.0
 
   # Install NerdBank GitVersioning
   - task: DotNetCoreCLI@2
     displayName: Install NBGV tool
     inputs:
       command: custom
       custom: tool
-      arguments: install --tool-path . nbgv
+      arguments: install -g nbgv
+
+  # Set Build Version
   - script: nbgv cloud
     displayName: Set NBGV version
 
   # Verify headers
-  - powershell: .\build\build.ps1 -Target Verify
+  - pwsh: build/Update-Headers.ps1 -Verify
     displayName: Verify headers
 
   # Build solution
-  - powershell: dotnet build -c Release
+  - script: dotnet build -c Release
```

Comment:
Why the change from `powershell` to `script` here?

Reply:
`pwsh` runs in `script` but `powershell` doesn't. Just avoiding pipeline overhead. We can use `pwsh` but ehh... it's simple, predictable and it works, at least for simple invocations.